core/src/main/java/org/infinispan/remoting/rpc/RpcManager.java

```diff
@@ -114,7 +216,9 @@
     * @param deliverOrder  the {@link org.infinispan.remoting.inboundhandler.DeliverOrder}.
     * @return a new {@link RpcOptionsBuilder} with the default options and the response mode and deliver mode set by the
     * parameters.
+    * @deprecated Since 9.2, please use {@link #getSyncRpcOptions()} instead.
     */
+   @Deprecated
    RpcOptionsBuilder getRpcOptionsBuilder(ResponseMode responseMode, DeliverOrder deliverOrder);
```

Comment:
should the `RpcOptionsBuilder` be deprecated also? Doesn't seems to be used anywhere since you are using the `RpcOption` directly.